util/getFullName.js

```diff
@@ -0,0 +1,10 @@
+export default function getFullName(firstName, lastName) {
+  let returnedName = '';
+
+  if (firstName && lastName) returnedName = `${firstName} ${lastName}`;
+  if (firstName && !lastName) returnedName = firstName;
+  if (!firstName && lastName) returnedName = lastName;
+  if (!firstName && !lastName) returnedName = 'Stranger';
```

Comment:
I also tried a `switch` statement, but decided that the `if`s read easier.

Reply:
We can take it a step further by returning immediately when a condition is true. That saves a bit of code execution and I find it a little easier to read.

```
if (firstName && lastName) return `${firstName} ${lastName}`;

// if we get here, we know at least one of the above was falsy
if (firstName) return firstName;

if (lastName) return lastName;

return 'Stranger';
```

Reply:
Gold!